src/test/mod.rs

```diff
@@ -136,6 +136,41 @@ fn basic_where_clauses_fail() {
 }
 
 #[test]
+fn basic_adt_variant_dup() {
+    crate::assert_err!(
+        [
+            crate Foo {
+                enum Bar {
+                    Baz{},
+                    Baz{},
+                }
+            }
+        ]
+
+        [ /* TODO */ ]
```

Comment:
add suggested messages in here

Reply:
ah wait, maybe instead it's some relevant substring in the error output that shouldn't change even when you update the expect output?